### PR TITLE
Add audit_enabled option to master_logging variable

### PR DESCRIPTION
**Pull request by stickyb1t**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -96,6 +96,7 @@ resource "yandex_kubernetes_cluster" "kube_cluster" {
       kube_apiserver_enabled     = var.master_logging.enabled_kube_apiserver
       cluster_autoscaler_enabled = var.master_logging.enabled_autoscaler
       events_enabled             = var.master_logging.enabled_events
+      audit_enabled              = var.master_logging.audit_enabled
```
Comment (pauljamm) [inline main.tf:99]: Please rename the option according to other logging options naming
var.master_logging.audit_enabled -> enabled_audit

File: variables.tf [hunk 2]
```diff
@@ -204,6 +204,7 @@ variable "master_logging" {
     enabled_kube_apiserver = true
     enabled_autoscaler     = true
     enabled_events         = true
+    audit_enabled          = false
```
Comment (pauljamm) [inline variables.tf:207]: the same here

**Review by pauljamm — CHANGES_REQUESTED** [review]
Hi!
Thanks for your PR.
Please change naming of the variable and regenerate docs.
Everything else is good to go

File: main.tf (hunk 1 above)
Comment (stickyb1t) [reply main.tf]: Hi, done

**Review by pauljamm — APPROVED** [review]

**Pull request merged**